lib/tasks_private/vpc.rake

```diff
@@ -0,0 +1,168 @@
+namespace :spec do
+  require 'ibm_cloud_sdk_core'
+  require 'ibm_cloud_resource_controller'
+  require 'ibm_vpc'
+
+  base_dir = File.join(ManageIQ::Providers::IbmCloud::Engine.config.root.to_s, 'spec')
+  cass_dir = File.join(base_dir, 'vcr_cassettes/manageiq/providers/ibm_cloud')
+  spec_dir = File.join(base_dir, 'models/manageiq/providers/ibm_cloud')
+
+  # Prerequisites:
+  # 1. Log in with IBM Cloud CLI i.e. `ibmcloud login -sso`
+  # 2. Place IBM Cloud API key in secrets config
+  desc "Generate refresher VCR"
+  task :refresh_init => :environment do
+    Dir["#{cass_dir}/vpc/cloud_manager/refresher*.yml"].each do |file|
+      FileUtils.rm(file, :verbose => true)
+    end
+    Dir["#{cass_dir}/vpc/cloud_manager/refresher/vm_target.yml"].each do |file|
+      FileUtils.rm(file, :verbose => true)
+    end
+
+    apikey = YAML.load_file("config/secrets.yml")["test"]["ibm_cloud_vpc"]["api_key"]
```

Comment:
Maybe `.dig("test", "ibm_cloud_vpc", "api_key")` and raise a helpful error if it is `nil`? (if a key in the path is missing this will just raise a `NoMethodError` exception